src/elements/content-sidebar/versions/VersionsSidebarContainer.js

```diff
@@ -292,6 +365,8 @@ class VersionsSidebarContainer extends React.Component<Props, State> {
                 onRestore={this.handleActionRestore}
                 onUpgradeClick={onUpgradeClick}
                 parentName={parentName}
+                showUpsell={showUpsell}
+                showUpsellWithPicture={showUpsellWithPicture}
```

Comment:
Can we eliminate the need for these props? Can we use the existence of `onUpgradeClick`, instead?